### PR TITLE
Update README.md

### LINKED ISSUE #55: Improve `homekit` instructions to use the integration directly from there
**Is your feature request related to a problem? Please describe.**

Add configuration instructions to the `README.md`.

**Describe the solution you'd like**

n/a

**Describe alternatives you've considered**

n/a

**Additional context**

n/a

### PR DESCRIPTION
**Pull request by syth87**
### Related Issues

- Closes #55 

### Proposed Changes:

Update `README.md` with Homekit configuration

### Testing:

n/a

### Extra Notes (optional):

n/a

### Checklist

- [x] Related issues and proposed changes are filled
- [ ] Tests are defining the correct and expected behavior
- [x] Code is well-documented via docstrings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,31 @@ UI example:
...
+
+1) Add these entries inside configuration.yaml to let Home Assistant create a new bridge with just the Alarm Panel exposed.
+
+```
+homekit:
+  - filter:
+      include_domains:
+        - alarm_control_panel
+    entity_config:
+      alarm_control_panel.alarm_panel:
```
Comment (palazzem) [inline README.md:105]: We don't need to make any change, but with the next release this name is going to change (we'll have `econnect_metronet` namespace). Don't change anything right now, but let's not forget about it!

**Review by palazzem — CHANGES_REQUESTED** [review]
This is a great contribution! Just a few little changes and we're good to go!

**Review by syth87 — COMMENTED** [review]
Update few lines

File: README.md [hunk 2]
```diff
@@ -90,6 +90,31 @@ UI example:
...
+       code: <PIN CODE>
+  - name: HASS Bridge Alarm
+    port: 21065
+```
+
+2) Please replace <PIN CODE> with your alarm code.
+3) Reboot Home Assistant.
+4) Scan the QR code inside the Notifications area with your iPhone to add the alarm into the Apple Home app.
```
[the remark this answers is not in the document]
Comment (syth87) [reply README.md]: Sure, changed!

File: README.md (hunk 1 above)
Comment (syth87) [reply README.md:105]: Domains section should remain the same but let's keep on eye on it.

File: README.md [hunk 3]
```diff
@@ -90,6 +90,31 @@ UI example:
 
 <img src="https://user-images.githubusercontent.com/1560405/110870435-ba91f900-82cc-11eb-9ebc-442152fd67f1.png" width="500"/>
 
+### HomeKit integration
+
+If you want to integrate your alarm with the Apple ecosystem to use Siri or Apple Home automations follow these steps:
+
+1) Add these entries inside configuration.yaml to let Home Assistant create a new bridge with just the Alarm Panel exposed.
```
[the remark this answers is not in the document]
Comment (syth87) [reply README.md]: Thanks

**Review by palazzem — APPROVED** [review]
This is great! thank you so much for your help!

**Pull request merged**